core/states/approaching_marker.py

```diff
@@ -1,10 +1,11 @@
 import asyncio
-from core.vision.ar_tag_detector import is_ar_tag
+from core.vision.ar_tag_detector import is_marker
 import core
 import interfaces
 import algorithms
 from core.states import RoverState
 import time
+import math
```

Comment:
Revisit this and self.gate_detection_attempts and other changes to see if unnecessary temporary changes are still lingering